treeherder/model/fixtures/repository.json

```diff
@@ -1110,20 +1110,6 @@
     }
 },
 {
-    "pk": 87,
-    "model": "model.repository",
-    "fields": {
-        "dvcs_type": "git",
-        "name": "addon-tests",
-        "url": "https://github.com/mozilla/Addon-Tests",
-        "branch": "master",
-        "active_status": "active",
```

Comment:
To disable repos, we set the `active_status` property to `onhold`, which will cause the existing DB entry to be updated. Removing the fixture wholesale would leave the prod DB entry orphaned. Ideally we'll have a better system for this at some point (plus we don't have a full cleanup command yet: https://bugzilla.mozilla.org/show_bug.cgi?id=1035294).

Reply:
Thanks @edmorley, I've updated the pull request.